root/etc/cont-init.d/50-config

```diff
@@ -8,40 +8,63 @@ mkdir -p \
 	/config/www/assets
 
 # check for .env and copy default if needed
-[[ ! -f "/config/www/config.inc.php" ]] && \
+[[ ! -f "/config/www/config.inc.php" ]] &&
 	cp /var/www/html/config.inc.php-sample /config/www/config.inc.php
 
 # check for zero-length .env and alert user if found
-[[ ! -s "/config/www/config.inc.php" ]] && \
-  echo "WARNING: zero-length config.inc.php file detected. Please delete /config/www/config.inc.php and restart the container"
+[[ ! -s "/config/www/config.inc.php" ]] &&
+	echo "WARNING: zero-length config.inc.php file detected. Please delete /config/www/config.inc.php and restart the container"
 
 # link assets
 rm -rf /var/www/html/assets
 ln -s /config/www/assets /var/www/html/assets
 
+# Set required variables depending on the db system used
```

Comment:
Please add mysql as a default value, so it also works when DB_SYSTEM is not set.

Reply:
Thanks for the remarks. I have some wondering around that, my assumption was that MRBS_DB_SYSTEM environment variable has to be set when launching the pod. Doing it here, without having set it up when launching the pod, then the variable MRBS_DB_SYSTEM is not set system-wide and then need to also be defined in the config.inc.php file.

What it could be done IMO would be to always define the variable in the docker-compose. If not then we need the fallback default value in the php config file or define it at php-fpm engine level in a `/etc/php81/php-fpm.d/mrbs.conf` file.

I have to push a commit with README instruction updated also.

Reply:
I think, the init scripts are sourced. So if you export something there, it should be exported in the later php-fpm process.

Reply:
Oh, nevermind, that's not the case. But we could just put `ENV MRBS_DB_SYSTEM=mysql` into the Dockerfile, that should do the trick.

Reply:
Yep, good catch let's go for that then